diff --git a/crates/storage/libmdbx-rs/Cargo.toml b/crates/storage/libmdbx-rs/Cargo.toml
index 2330b6f79..d6eee8804 100644
--- a/crates/storage/libmdbx-rs/Cargo.toml
+++ b/crates/storage/libmdbx-rs/Cargo.toml
@@ -28,7 +28,7 @@ tracing.workspace = true
 ffi = { package = "reth-mdbx-sys", path = "./mdbx-sys" }
 
 [target.'cfg(not(windows))'.dependencies]
-libffi = "3.2.0"
+libffi = { version = "3.2.0", features = ["system"] }
 
 [features]
 default = []
